assets/css/v2/on_bus/time.scss

```diff
@@ -0,0 +1,44 @@
+.departure-time {
+  display: inline-block;
+  align-items: center;
+  justify-content: end;
```

Comment:
Since this is not a flex or grid container I don't think `align-items` and `justify-content` would have any effect?

Reply:
oh yes those are leftover from when I was using a flex container here!